src/virtualenv/util/lock.py

```diff
@@ -25,10 +25,10 @@ def __init__(self, lock_file):
         self.count = 0
         self.thread_safe = RLock()
 
-    def acquire(self, timeout=None, poll_intervall=0.05):
+    def acquire(self, timeout=None, poll_interval=0.05):
         with self.thread_safe:
             if self.count == 0:
-                super(_CountedFileLock, self).acquire(timeout=timeout, poll_intervall=poll_intervall)
+                super(_CountedFileLock, self).acquire(timeout=timeout, poll_interval=poll_interval)
```

Comment:
```suggestion
                super(_CountedFileLock, self).acquire(timeout, poll_interval)
```

Since we cannot use `filelock >= 3.4` (see comment above), use positional arguments.

Reply:
Thanks!